modules/images-configuration-allowed.adoc

```diff
@@ -12,7 +12,7 @@ When pulling or pushing images, the container runtime searches the registries li
 
 [WARNING]
 ====
-When the `allowedRegistries` parameter is defined, all registries including the `registry.redhat.io` and `quay.io` registries are blocked unless explicitly listed. If you use the parameter, to prevent pod failure, add `registry.redhat.io` and `quay.io` to the `allowedRegistries` list, as they are required by payload images within your environment. For disconnected clusters, mirror registries should also be added.
+When the `allowedRegistries` parameter is defined, all registries including the `registry.redhat.io` and `quay.io` registries are blocked unless explicitly listed. If you use the parameter, to prevent pod failure, you must add `registry.redhat.io` and `quay.io` to the `allowedRegistries` list, as they are required by payload images within your environment. For disconnected clusters, mirror registries should also be added.
```

Comment:
Not in scope for this PR but this part is a bit unclear:
> all registries including the `registry.redhat.io` and `quay.io` registries are blocked

I see 2 interpretations: 

- > all registries that include the `registry.redhat.io` and `quay.io` registries are blocked
- > all registries, including `registry.redhat.io` and `quay.io` registries, are blocked

Guessing the second option is the correct one because for the first option it probably wouldn't make sense to have a registry include a registry.

If you agree, maybe you could comma-tize that part for clarity?